toolboxes/mri_core/mri_core_export.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _MRI_CORE_EXPORT_H_
+#define _MRI_CORE_EXPORT_H_
+
+#if defined (WIN32)
+    #ifdef BUILD_TOOLBOX_STATIC
```

Comment:
I want this static toolbox stuff gone. It messes with a lot of stuff, specifically the ability to reliably count on singletons across platforms.